docs/key-formats.md

```diff
@@ -83,6 +84,8 @@ If `format` is missing, the wallet uses a default of bip32.
 
 If the `coinType` is missing, the default is 0 for Bitcoin. Other coins use different defaults for `coinType`.
 
+The `imported` field signifies whether the wallet was imported or created. If the `imported` field is `undefined`, then the wallet must have been created before this field was added to the specification and whether the wallet was imported or created is unknown. Implementations should specify `true` if the wallet was imported, and `false` if the wallet was created moving forward.
+
```

Comment:
I would word this differently:

> The `imported` field should be `true` for keys that the user imported, and `false` for keys that the app generated randomly. The `imported` field is optional, and if it is missing, the keys were created before the app began recording this information.

Also, I would move this paragraph to the "Storage keys" section, which should then be renamed to "Common key properties" or such.